src/constants.ts

```diff
@@ -22,7 +22,7 @@ export const FIAT_DISPLAY_PRECISION = 2;
 
 export const TOKEN_DISPLAY_PRECISION = 6;
 
-export const SNACKBAR_AUTO_HIDE_DURATION = 5000;
+export const SNACKBAR_AUTO_HIDE_DURATION = 8000;
```

Comment:
I wouldn't change this value, 8s is too much for a snackbar

Reply:
Even with 8 seconds, [we're violating the WCAG requirement](https://www.w3.org/TR/WCAG21/#enough-time) related to time-based operation from an accessibility perspective.

IMO with 5s, it's difficult to read, considering I don't have any visual or (diagnosed) cognitive issues, and I know when and where the snackbar will appear. I feel a user needs 2s just to cognitively context-switch and start reading the text, which leaves 3 seconds for the content.

Anyway, this is not an exact science. I will revert it but wanted to share my reasoning to increase it in the first place.

Reply:
Reverted.